### ISSUE TITLE
Node upgrade / deprecation warnings

### ISSUE THREAD

**Issue by alecglen** [githubocto/flat#76]
Getting these in the action summary:

![image](https://user-images.githubusercontent.com/28493454/195734680-67047718-c94c-4702-a934-0ff64b3415a6.png)

**Comment by akosiaris** [comment]
I am meeting this too. I see that the last commit in this repo is in Aug 2021. Is this action still maintained?

**Comment by aslak01** [comment]
I'm also seeing these warnings. Hope it'll be resolve. Node LTS is already on v18 👀

**Comment by skhalaf21** [comment]
Any word on this? are there any plans to patch this action so it does not throw this Node 12 warning and eventually stop working?

**Comment by alecglen** [comment]
@idan , @Wattenberger , @mattrothenberg , Bueller...

**Comment by akosiaris** [comment]
@mattrothenberg 

Hi, thanks for your work on this. I have switched to using the action you suggested today. It fixes the issue reported here and also works fine for my use case. I 'll let it run (it runs on a schedule) for a couple of weeks and report back here.

**Comment by mattrothenberg** [comment]
This should hopefully be fixed in https://github.com/githubocto/flat/releases/tag/3.4.0 🤞

**Comment by alecglen** [comment]
Much appreciated, @mattrothenberg!

I think this is working for me too, but there's something I'm missing. Hopefully the gang assembled here can help make it make sense?

I haven't updated my my flat.yaml since the release, but I checked just now and the deprecation warnings are indeed gone as of a couple weeks ago. I'm confused because from what I can tell in the action logs, it's still running with an older version (3.3.1 based on the SHA).
```
...
Download action repository 'githubocto/flat@v3' (SHA:fc34373cc20b987e89d75d96435a9fecd8fb0807)
...
```

However, the warnings are definitely gone, so I'm not sure what to believe :shrug: . Is there anything I need to do explicitly in the yaml to make sure I'm on the new release?

**Comment by mattrothenberg** [comment]
@alecglen That sounds weird! Are you able to link to the particular action run where you're seeing that?

**Comment by alecglen** [comment]
Yeah I can ...it's just in a private repo that isn't actually top-secret haha. Just sent you a collaborator invite so you can view it.

**Comment by mattrothenberg** [comment]
Great, will take a look tomorrow!

**Comment by akosiaris** [comment]
> @mattrothenberg
> 
> Hi, thanks for your work on this. I have switched to using the action you suggested today. It fixes the issue reported here and also works fine for my use case. I 'll let it run (it runs on a schedule) for a couple of weeks and report back here.

Couple of weeks passed, not a single error for me. Many thanks again!

**Comment by dalanmiller** [comment]
Still getting this on `flat@v3` unfortunately? Not sure why.

**Comment by skhalaf21** [comment]
@dalanmiller  have you tried with `githubocto/flat@v3.4.0` ?

**Comment by dalanmiller** [comment]
Ahhhh wonderful that was it. 

I assumed incorrectly that using `@v3` would grab the latest minor point release.

**Comment by skhalaf21** [comment]
@dalanmiller no problem. Actually, that's what I also though, but apparently it was not the case.
